gomint-api/src/main/java/io/gomint/scheduler/WorldScheduler.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2020, GoMint, BlackyPaw and geNAZt
+ *
+ * This code is licensed under the BSD license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package io.gomint.scheduler;
+
+import io.gomint.world.World;
+
+import java.util.concurrent.TimeUnit;
+
+/**
+ * @author Janmm14
+ * @version 2.0
+ * @stability 2
+ */
+public interface WorldScheduler extends AsyncScheduler {
+
+    /**
+     * Gets the world this scheduler will submit synchronized tasks to
+     *
+     * @return the world synchronized tasks will be associated to
+     */
+    World world();
```

Comment:
This can be null, please document

Reply:
When can this be null?
When plugins for whatever reason save WorldScheduler instances forever and access it after the world was unloaded?
Then the whole WorldScheduler instance gets invalid and useless.

Reply:
Yes thats the case. We should document that this will become null when the world unloads and thus storing a instance of this scheduler is NOT recommended

Reply:
documented